pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+#
+# This script performs:
+# - segmentation of rootlets from T2w data (model model-spinal-rootlets_ventral_D106_r20240523),
+# - segmentation of spinal cord from T2w data (sct_deepseg_sc)
+# - segmentation of vertebral levels from T2w data (sct_label_vertebrae)
+# - detection of PMJ from T2w data (sct_detect_pmj).
+#
+# Authors: Katerina Krejci, Jan Valosek
+#
+
+# Uncomment for full verbose
+set -x
+
+# Immediately exit if error
+set -e -o pipefail
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+# Retrieve input params
+SUBJECT=$1    # sub-0001/ses-01
+
+# get starting time:
+start=`date +%s`
+# FUNCTIONS
+# ==============================================================================
+
+# Segment spinal cord if it does not exist
+segment_if_does_not_exist(){
+  FILESEG="${file_t2}_label-SC_mask"
+  FILESEGMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILESEG}.nii.gz"
+  echo
+  echo "Looking for manual segmentation: $FILESEGMANUAL"
+  if [[ -e $FILESEGMANUAL ]]; then
+    echo "Found! Using manual segmentation."
+    rsync -avzh $FILESEGMANUAL ${FILESEG}.nii.gz
+    sct_qc -i ${file_t2}.nii.gz -s ${FILESEG}.nii.gz -p sct_deepseg_sc -qc ${PATH_QC} -qc-subject ${SUBJECT}
+  else
+    echo "Not found. Proceeding with automatic segmentation."
+    # Segment spinal cord
+    sct_deepseg_sc -i ${file_t2}.nii.gz -c t2 -qc ${PATH_QC} -qc-subject ${SUBJECT} -o ${FILESEG}.nii.gz
+  fi
+}
+
+# Detect PMJ if it does not exist
+detect_pmj_if_does_not_exist(){
+  FILEPMJ="${file_t2}_label-PMJ_dlabel"
+  FILEPMJMANUAL="${PATH_DATA}/derivatives/labels/${SUBJECT}/anat/${FILEPMJ}.nii.gz"
+  echo
+  echo "Looking for manual PMJ detection: $FILEPMJMANUAL"
+  if [[ -e $FILEPMJMANUAL ]]; then
+    echo "Found! Using manual PMJ detection."
+    rsync -avzh $FILEPMJMANUAL ${FILEPMJ}.nii.gz
+    sct_qc -i ${file_t2}.nii.gz -s ${FILESEG}.nii.gz -d ${FILEPMJ}.nii.gz -p sct_detect_pmj -qc ${PATH_QC} -qc-subject ${SUBJECT}
```

Comment:
`sct_qc -p sct_detect_pmj` expects only `-s`, not `-d`; see [here](https://github.com/sct-pipeline/pmj-based-csa/blob/419ece49c81782f23405d89c7b4b15d8e03ed4bd/identify_nerve.sh#L65)
```suggestion
    sct_qc -i ${file_t2}.nii.gz -s ${FILEPMJ}.nii.gz -p sct_detect_pmj -qc ${PATH_QC} -qc-subject ${SUBJECT}
```